api/pom.xml

```diff
@@ -21,7 +21,7 @@
     <parent>
         <groupId>com.okta.sdk</groupId>
         <artifactId>okta-sdk-root</artifactId>
-        <version>4.1.1-SNAPSHOT</version>
+        <version>5.0.0-SNAPSHOT</version>
```

Comment:
I believe, this PR contains many method additions to existing interfaces. This would be a `MINOR` vesion upgrade according to [semver](https://siom79.github.io/japicmp/MavenPlugin.html).

Reply:
This build also has `METHOD_REMOVED`  and `METHOD_RETURN_TYPE_CHANGED`  changes which require major version increase.

`METHOD_REMOVED` in :

- com.okta.sdk.resource.authorization.server.policy.AuthorizationServerPolicyRule
- com.okta.sdk.resource.user.schema.UserSchemaAttribute
- com.okta.sdk.resource.user.schema.UserSchemaAttributeMaster

`METHOD_RETURN_TYPE_CHANGED` in :
- com.okta.sdk.resource.user.schema.UserSchemaAttributeMaster
- com.okta.sdk.resource.user.schema.UserSchema
- com.okta.sdk.resource.user.schema.UserSchemaAttribute